yarn.lock

```diff
@@ -2017,6 +2039,12 @@ hash.js@^1.0.0, hash.js@^1.0.3, hash.js@^1.1.7:
     inherits "^2.0.3"
     minimalistic-assert "^1.0.1"
 
+"hashi-vault-js@git+https://github.com/josedev-union/hashi-vault-js":
```

Comment:
We can use my personal git repo till this PR is merged, 
https://github.com/rod4n4m1/hashi-vault-js/pull/26